spras/local_neighborhood.py

```diff
@@ -0,0 +1,136 @@
+from spras.prm import PRM
+from pathlib import Path
+from spras.containers import prepare_volume, run_container
+from spras.util import add_rank_column
+import pandas as pd
+from spras.interactome import reinsert_direction_col_undirected
+
+__all__ = ['LocalNeighborhood']
+
+
+class LocalNeighborhood(PRM):
+    required_inputs = ['network', 'nodetypes']
```

Comment:
PathLinker has a "nodetypes" file because it had sources and targets. Local neighborhood only has one "type" of node, so we could consider calling this input file something different.